static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate-copy-test-variant.js

```diff
@@ -0,0 +1,26 @@
+export const signInGateCopyTest = {
+	id: 'SignInGateCopyTest',
+	start: '2023-01-23',
+	expiry: '2025-12-01',
+	author: 'Lindsey Dew',
+	description: 'Test varying the copy in the call to action for sign in gate',
+	audience: 0.1,
+	audienceOffset: 0.0,
+	successMeasure: 'Users sign in or create a Guardian account',
+	audienceCriteria: '3rd article of the day, lower priority than consent banner, simple articles (not gallery, live etc.), not signed in, not shown after dismiss, not on help, info sections etc. Exclude iOS 9 and guardian-live-australia. Suppresses other banners, and appears over epics',
+	dataLinkNames: 'SignInGateCopyTest',
+	idealOutcome:
```

Comment:
We should ideally be able to add some firmer result numbers in ideal outcome as part of the test hypothesis once we get that info from D&I

Reply:
Updated this from the doc: https://docs.google.com/document/d/1lBufse0NxTZqRNFFRbch1hcFlyv65gEyNyMHtoqKj48/edit